src/index.js

```diff
@@ -90,8 +90,9 @@ const sendXmlHttpRequest = (path, method, headers, body) => {
   })
 }
 
-const sendNodeAPIRequest = (path, method, headers, body) => {
+const sendNodeAPIRequest = (path, method, headers, body, encoding) => {
   return new Promise((resolve, reject) => {
+    const Buffer = require('buffer').Buffer
```

Comment:
[Buffer](https://nodejs.org/api/globals.html#globals_class_buffer) is available in node.js without 'requiring' it